composer.json

```diff
@@ -25,7 +25,8 @@
     "require": {
         "php": "^8.0",
         "illuminate/support": "^8.24",
-        "imliam/laravel-blade-helper": "^1.0"
+        "imliam/laravel-blade-helper": "^1.0",
+        "intervention/image": "^2.7"
```

Comment:
I think we could remove this dependency and make it an optional add-on, since this probably adds a lot of size to the package. See the other review.